SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunner/String.swift

```diff
@@ -10,3 +10,7 @@ import Foundation
 func create_swift_string() -> String {
     "hello"
 }
+
+func take_and_return_swift_string(arg: RustString) -> String {
+    arg.toString()
+}
```

Comment:
@chinedufn 

`create_swift_string` returns `String`. To maintain consistency and remove `toRustString`(`extern_swift_func_takes_and_returns_string`), I changed `take_and_return_swift_string` return value from `RustString` to `String`. But it is inefficient. what do you think about this?

Reply:
Here're `create_swift_string` and `take_and_return_swift_string` implementation.
```String
//
//  String.swift
//  SwiftRustIntegrationTestRunner
//
//  Created by Frankie Nwafili on 2/18/22.
//

import Foundation

func create_swift_string() -> String {
    "hello"
}

func take_and_return_swift_string(arg: RustString) -> String {
    arg.toString()
}

```

Reply:
I haven't reviewed the code yet but this sounds fine.

Let's just document this behavior in the `String` chapter in the book so that users are aware of this implicit clone